app/src/main/res/xml/keys_letters_chuvash.xml

```diff
@@ -0,0 +1,195 @@
+            app:topSmallNumber="1" />
+        <Key
+            app:keyLabel="ӑ"
+            app:popupCharacters="2"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="2" />
+        <Key
+            app:keyLabel="ӗ"
+            app:popupCharacters="3"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="3" />
+        <Key
+            app:keyLabel="ҫ"
+            app:popupCharacters="4"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="4" />
+        <Key
+            app:keyLabel="ӳ"
+            app:popupCharacters="5"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="5" />
+        <Key
+            app:keyLabel="ъ"
+            app:popupCharacters="6"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="6" />
+        <Key
+            app:keyLabel="-"
+            app:popupCharacters="7"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="7" />
+        <Key
+            app:keyLabel="!"
+            app:popupCharacters="8"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="8" />
+        <Key
+            app:keyLabel="\?"
+            app:popupCharacters="9"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="9" />
+        <Key
+            app:keyEdgeFlags="right"
+            app:keyLabel="&quot;"
+            app:popupCharacters="«»„“0"
+            app:popupKeyboard="@xml/keyboard_popup_template"
+            app:topSmallNumber="0" />
+    </Row>
+    <Row app:keyWidth="9.05%p">
+        <Key
+            app:keyEdgeFlags="left"
+            app:keyLabel="й" />
+        <Key app:keyLabel="ц" />
+        <Key app:keyLabel="у" />
+        <Key app:keyLabel="к" />
+        <Key app:keyLabel="е" />
+        <Key app:keyLabel="н" />
+        <Key app:keyLabel="г" />
+        <Key app:keyLabel="ш" />
+        <Key app:keyLabel="щ" />
+        <Key app:keyLabel="з" />
+        <Key
+            app:keyEdgeFlags="right"
+            app:keyLabel="х" />
+    </Row>
+    <Row app:keyWidth="9.05%p">
+        <Key
+            app:keyEdgeFlags="left"
+            app:keyLabel="ф" />
+        <Key app:keyLabel="ы" />
+        <Key app:keyLabel="в" />
+        <Key app:keyLabel="а" />
+        <Key app:keyLabel="п" />
+        <Key app:keyLabel="р" />
+        <Key app:keyLabel="о" />
+        <Key app:keyLabel="л" />
+        <Key app:keyLabel="д" />
+        <Key app:keyLabel="ж" />
+        <Key
+            app:keyEdgeFlags="right"
+            app:keyLabel="э" />
+    </Row>
+    <Row>
+        <Key
+            app:code="-1"
+            app:keyEdgeFlags="left"
+            app:keyIcon="@drawable/ic_caps_outline_vector"
+            app:keyWidth="9.275%p" />
```

Comment:
Shift and Backspace should use 15% of the available space as done in other layouts.